### DISCUSSION TITLE
文档中指明了在streamload时，列转换支持UDF，但实际使用中报Does not support non-builtin functions, or function does not exist

### DISCUSSION THREAD

**Discussion by swiftlei** [apache/doris#41214] [A - General / Q&A]
DORIS版本: 2.1.6

官方文档中说在 streamload 的列转换时支持使用udf，但在streamload中，使用udf会报Does not support non-builtin functions, or function does not exist的错误，具体运行条件如下：

1、http put streamload + basicauth方式，用户为root权限；
2、数据格式为 json (read_json_by_line = true 的模式)
3、在 header中增加了  columns: field1, field2, field3 ..., field4 = myudf1(field1), field5 = myudf2(field1)

如果把 myudf1， myudf2 更换为系统内置函数，如 lower()，则运行正常，但如使用用户自定义函数则出现上述错误。

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。
